argoverse/evaluation/eval_detection.py

```diff
@@ -0,0 +1,410 @@
+# <Copyright 2020, Argo AI, LLC. Released under the MIT license.>
+"""Detection evaluation for the Argoverse detection leaderboard.
+
+Evaluation:
+
+    Precision/Recall
+
+        1. Average Precision: Standard VOC-style average precision calculation
+            except a true positive requires a bird's eye center distance of less
+            than a predefined threshold.
+
+    True Positive Errors
+
+        All true positive errors -- as the name implies -- accumulate error solely
+        when an object is a true positive match to a ground truth detection. The matching
+        criterion is represented by `tp_thresh` in the DetectionCfg class.
```

Comment:
i would mention that we only compute AOE, ATE, ASE @2 m thresh, and that's what we will abide by